app/workers/terms_of_service_update_worker.rb

```diff
@@ -4,17 +4,19 @@ class TermsOfServiceUpdateWorker
   sidekiq_options queue: 'terms_mailer', retry: 0
 
   def perform
-    last_updated = Time.at(User.terms_last_updated_at)
-    updated_time = Time.now
-    # Based on our AWS SES account (Maximum send rate: 200 emails per second) I set a batch size 200 and do a sleep 1
-    User.where(is_active: true).where('email IS NOT NULL')
-    .where('last_received_terms_email_at < ?', last_updated)
-    .find_in_batches(:batch_size => 200) do |users|
-      users.each do |u|
-        UpdatedTermsMailer.delay({ retry: 1, queue: 'terms_mailer' }).notify(u.email, u.name)
-        u.update_columns(last_received_terms_email_at: updated_time)
+    if Rails.cache.read('enable_terms_last_updated_at_notification')
+      last_updated = Time.at(User.terms_last_updated_at)
```

Comment:
@melsawy, by reusing this flag, there is no way to just send the email without showing the dialog in Check Web, since the email also relies in `terms_last_updated_at`, right? We should be able to send the email without necessarily showing the dialog. So, I suggest we have another flag for that.